rust/kernel/print.rs

```diff
@@ -132,6 +138,9 @@ pub unsafe fn call_printk(
     args: fmt::Arguments<'_>,
 ) {
     // `_printk` does not seem to fail in any path.
+    // SAFETY: references are guaranteed to be valid for reads.
+    // The caller guarantees that `format_string` is a valid format
+    // specifier and that `module_name` is null-terminated.
```

Comment:
This is fine, although in general I try to avoid repeating the contracts and/or invariants.

We could instead take the chance to explain that `printk` reads up to a NULL, which is guaranteed to be in our slice.

But do not worry too much, it is understandable either way.